.github/workflows/deps-ubuntu-24.04-gcc.txt

```diff
@@ -0,0 +1,51 @@
+binutils
+ca-certificates
+ccache
+cmake
+file
+fluid
+gcc
+git
+gpg
+g++
+libasound2-dev
+libc6-dev
+libfftw3-dev
+libfltk1.3-dev
+libfluidsynth-dev
+libgig-dev
+libgtk2.0-0
+libjack-jackd2-dev
+liblilv-dev
+liblist-moreutils-perl
+libmp3lame-dev
+libogg-dev
+libqt5x11extras5-dev
```

Comment:
```suggestion
libqt5svg5-dev
libqt5x11extras5-dev
```
Add Qt5 SVG library to fix Linux ARM64 build

Reply:
Good catch.